.github/workflows/ubuntu.yml

```diff
@@ -16,14 +16,12 @@ jobs:
       uses: actions/setup-python@v3
       with:
         python-version: "3.8"
-    - name: Install dependencies
+    - name: Install linters
       run: |
         python -m pip install --upgrade pip
         pip install -r linters-requirements.txt 
-        pip install .
-    - name: Lint with flake8
-      run: |
-        flake8 --show-source
+    - name: Lint with OpenFL-specific rules
+      run: bash shell/lint.sh
```

Comment:
@teoparvanov Not sure I understood why do we need a flake8 test in Ubuntu workflow?
I do see that it existed before albeit without correct args, but I don't think we need it here in the first place (it is already covered in lint test of every PR)

Reply:
Hi @MasterSkepticista, in terms of integration, I think there is value in doing testing post-merge, just to confirm that the latest code is stable. I agree this is less relevant in the linting case, as any regressions would be caught at the PR level. Still, I'd prefer to keep the existing pattern, just aligning it with the rules defined for the PR workflow.

Reply:
PS: as you suggested, I'll drop the code fixes which are not needed given the settings in `setup.cfg`